README.adoc

```diff
@@ -287,7 +312,7 @@ https://github.com/ChumbokIT[Github]
 
 |Integrated Software and Technologies Ltd.
 |Avenue # 9, Road #9, House #1043 (2nd Floor) DOHS Mirpur, Dhaka, Bangladesh
```

Comment:
Hashtag in address should be removed, as GitHub renders #number as a pull request link.

Reply:
Fixed